### ISSUE TITLE
WEBUI : Misleading imcompatible notice when no cluster is selected om simulation

### ISSUE THREAD

**Issue by MaxandreOgeret** [lgsvl/simulator#1563]
Hello,

When a simulation is not linked to any cluster (for example when the cluster is deleted) the simulation overview alerts will display misleading messages : 

![image](https://user-images.githubusercontent.com/22786578/124903858-10617e00-dfed-11eb-882f-183613e54ff1.png)

![image](https://user-images.githubusercontent.com/22786578/124903901-1c4d4000-dfed-11eb-9af7-1f6ef35fe7d0.png)

Those message shouldn't appear as they refer to the 'simulation cluster version' which doesn't exist. And how is it possible that it checks a cluster version if there is no cluster ?

For me the right solution would be to have a dedicated alert when no cluster is displayed that will prevent other alerts to be displayed.

Thanks !

**Comment by lemketron** [comment]
> For me the right solution would be to have a dedicated alert when no cluster is displayed that will prevent other alerts to be displayed.

You are correct! Fix the cluster first and the other messages will likely go away (and should not be presented as long as the cluster is invalid).

I encountered this same issue myself just this morning. We will investigate that.
